src/client/opamCommands.ml

```diff
@@ -308,7 +308,7 @@ let init cli =
       mk_vflag ~cli `none [
         cli_from ~experimental:true cli2_2,
         `internal, ["cygwin-internal-install"],
-        "Let opam setup and manage an internal Cygwin install (recommended)";
+        "Let opam setup and manage an internal Cygwin install";
```

Comment:
This should be kept

Reply:
I don't get the context. Why removing it here ?

Reply:
you're removing something (the ` (recommended)` part) for some reason and I think it should be kept as-is, as it was just added in https://github.com/ocaml/opam/pull/5903

Reply:
Oh sorry I missed that, I thought is your proposition to remove it. This was not intended, thanks.